tests/integration-tests/tests/performance_tests/test_starccm.py

```diff
@@ -0,0 +1,72 @@
+import pytest
+import logging
+import boto3
+import json
+
+from remote_command_executor import RemoteCommandExecutor, RemoteCommandExecutionError
+
+# timeout in seconds
+INSTALLER_TIMEOUT = 1800
+SCHEDULER_TIMEOUT = 600
+STARCCM_SECRET_NAME = 'perf_integ_secret'
+NODES = [8, 16, 32]
+
+# TODO: replace with 8,16,32 nodes, test with 2 compute nodes, injected into pcluster config reader
+
+
+def get_starccm_secrets(region_name):
+    secrets_manager_client = boto3.client('secretsmanager', region_name=region_name)
+    response = secrets_manager_client.get_secret_value(SecretId="arn:aws:secretsmanager:eu-west-1:815618621112:secret:perf_integ_secret-iZqSUK")["SecretString"]
```

Comment:
You already have a constant that specifies the name of the secret, let's make use of it.